ZFSin/zfs/module/zfs/zfs_windows_zvol_scsi.c

```diff
@@ -639,55 +597,49 @@ ScsiOpModeSense(
 /**************************************************************************************************/     
 /*                                                                                                */     
 /**************************************************************************************************/     
+VOID ReverseIndian(UCHAR *x)
+{
+	UCHAR t;
+	UCHAR* d = (UCHAR*)x;
+	t = d[0]; d[0] = d[3]; d[3] = t; t = d[1]; d[1] = d[2]; d[2] = t;
+}
+
```

Comment:
Is this supposed to be `ReverseEndian()` ? As in changing to BigEndian (presumably scsi operates on big)? Should we not use `htonl()` in that case?

Reply:
Yes, htonl() works but after checking the WDK I found that they recommend using the Rtl version of the WDK API: https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/wdm/nf-wdm-rtlulongbyteswap
So I used that, retested and committed. Thanks.

Reply:
Yeah it seems on windows htonl and family is in winsock2, so it is a whole thing, even if we are linking against it in this project. Thanks for the update.